poetry/core/masonry/builders/builder.py

```diff
@@ -61,11 +61,28 @@ def __init__(
 
             packages.append(p)
 
+        includes = []
+        for include in self._package.include:
+            if isinstance(include, dict):
```

Comment:
Should we consider moving this check into the factory implementation? So we can normalise access to includes? Same with formats.

Reply:
Yes, I think that's a good idea.